src/types.ts

```diff
@@ -0,0 +1,36 @@
+export interface ISale {
```

Comment:
As I said previously, I'd prefer put it on a sales.ts and isolate context

Reply:
We can do it!